### PR TITLE
Clarify the PHP version needed

### PR DESCRIPTION
**Pull request by spekulatius**
As mentioned in #33

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
     }
   ],
   "require": {
-    "php": "^7.2.5",
+    "php": "^7.3",
```
Comment (imami) [inline composer.json:24]:
```suggestion
    "php": "^7.3|^7.4",
```

**Comment by spekulatius** [comment]
Did you check if works on PHP8 @imami ?

**Comment by imami** [comment]
Not yet. @spekulatius

**Comment by sarthaksavvy** [comment]
Thanks done with this version

**Pull request closed without merging**